engine/flink/test-util/src/main/scala/pl/touk/nussknacker/engine/flink/test/ClassExtractionBaseTest.scala

```diff
@@ -28,7 +28,7 @@ trait ClassExtractionBaseTest extends FunSuite with Matchers {
 
     //printFoundClasses(types)
     //println(types.asJson.spaces2)
-    checkGeneratedClasses(types, decoded)
+    //checkGeneratedClasses(types, decoded)
```

Comment:
until we update allowed classes and methods snapshot for including arrays

Reply:
so it is still a draft?

Reply:
changed it back with updated classes and methods, unfortunately still not sure why it doesn't pass the test, parameters from methods extracted from file are parsed as unknown where parameters from model.processWithObjectsDefinition are known classes 
i will write update once i fixed it